cmd/list_components.go

```diff
@@ -30,26 +29,26 @@ var listComponentsCmd = &cobra.Command{
 
 		stackFlag, err := flags.GetString("stack")
 		if err != nil {
-			u.PrintMessageInColor(fmt.Sprintf("Error getting the 'stack' flag: %v", err), color.New(color.FgRed))
+			u.PrintErrorMarkdownAndExit("Invalid Usage", fmt.Errorf("Error getting the `stack` flag: `%v`", err), "")
```

Comment:
Maybe we shouldn't support custom titles, because then we get inconsistencies like this: 

```
u.PrintErrorMarkdownAndExit("Incorrect Usage", ...
```

Maybe we should have the commont types of errors.
- `u.InvalidUsage(fmt.Errorf("Error getting the `stack` flag: `%v`", err), nil)`
- `u.InvalidCommand(fmt.Errorf("Error getting the `stack` flag: `%v`", err), nil)`

Also, some of our titles are proper title case, while others are not.

Reply:
Done

Reply:
Done